### PR TITLE
Add e2e troubleshooting tip

### PR DESCRIPTION
**Pull request by yolocs**
A note on how to find logs dump.

## Proposed Changes

-
-
-

**Release Note**

```release-note
NONE
```

**Docs**

### CODE REVIEW

File: docs/testing/troubleshooting-e2e.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+prow links on the PR page. Such links are in the format of
+`https://prow.knative.dev/view/gcs/knative-prow/pr-logs/pull/google_knative-gcp/[PR ID]/[TEST NAME]/[TEST ID]` 
+, e.g.
+`https://prow.knative.dev/view/gcs/knative-prow/pr-logs/pull/google_knative-gcp/1153/pull-google-knative-gcp-integration-tests/1267481606424104960` 
+.
+
+If the prow page doesn't provide any useful information, check out the control
+plane logs dump (pods in `cloud-run-events` namespace). The logs dump are saved
+in a public GCS bucket. Find the logs at
+`https://console.cloud.google.com/storage/browser/knative-prow/pr-logs/pull/google_knative-gcp/[PR ID]/[TEST NAME]/[TEST ID]/artifacts/controller-logs/`
```
Comment (liu-cong) [inline docs/testing/troubleshooting-e2e.md:13]: Are these other logs as well in addition to the contoller-logs?

File: docs/testing/troubleshooting-e2e.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Troubleshooting E2E Tests
```
Comment (capri-xiyue) [inline docs/testing/troubleshooting-e2e.md:1]: Can you also help add the pod logs https://github.com/knative/eventing/pull/3113 and metric logs https://github.com/google/knative-gcp/pull/1069 in this troubleshooting session?

File: docs/testing/troubleshooting-e2e.md (hunk 1 above)
Comment (yolocs) [reply docs/testing/troubleshooting-e2e.md]: Added pod logs as suggested by Xiyue.

File: docs/testing/troubleshooting-e2e.md (hunk 2 above)
Comment (yolocs) [reply docs/testing/troubleshooting-e2e.md:1]: Added pod logs. The metric logs seem to be already in the prow page IIUC.

**Pull request merged**